src/main/java/telraam/database/models/Baton.java

```diff
@@ -1,16 +1,17 @@
 package telraam.database.models;
 
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
 import java.util.Objects;
 
+@Getter @Setter @NoArgsConstructor
 public class Baton {
     private Integer id;
     private String name;
     private String mac;
 
-    // DO NOT REMOVE
-    public Baton() {
-    }
```

Comment:
Also, rebase on development to get the latest CI change here?

Reply:
I've removed it because it is replaced with the `@NoArgsConstructor`. It think JDBI needs it to create the object after fetching it from the DB (and then changes the values via setters)